docs/2.guide/2.directory-structure/3.app-config.md

```diff
@@ -35,7 +35,11 @@ export default defineAppConfig({
 })
 ```
 
-When adding `theme` to the `app.config`, Nuxt uses Vite or webpack to bundle the code. We can universally access `theme` in both server and browser using [useAppConfig](/docs/api/composables/use-app-config) composable.
+When adding `theme` to the `app.config`, Nuxt uses Vite or webpack to bundle the code. We can universally access `theme` in both server-side rendering and client-side using [useAppConfig](/docs/api/composables/use-app-config) composable.
```

Comment:
I feel like I'm nitpicking now 😆 

```suggestion
When adding `theme` to the `app.config`, Nuxt uses Vite or webpack to bundle the code. We can universally access `theme` both when server-rendering the page and in the browser using [useAppConfig](/docs/api/composables/use-app-config) composable.
```

Reply:
Do you think we still need the alert now?

Reply:
The original text now clearly explains where the composable can be used. So, could totally remove the alert or make it talk about Nitro support coming soon (+10% clarity 😅).

Reply:
Great! Let's convert it to focus on Nitro support coming soon - we could even link to this issue instead: https://github.com/nuxt/nuxt/issues/14670